### PR TITLE
Newsweek clone

### PR DESCRIPTION
**Pull request by paulzay**
### What does this pr do?
Creates a clone project of newsweek.com
### Description of tasks to be completed?
1. Use bootstrap to create the project

### CODE REVIEW

**Comment by elmejdki** [comment]
### Pull Request under review by **Zakariae El Mejdki**

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
 # newsweek
-USING BOOTSTRAP
+
+>USING BOOTSTRAP
+
+
```
Comment (elmejdki) [inline README.md]: Please add a preview picture of your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1532 @@
...
+          </li>
+
+          <li class="nav-item">
+            <a class="nav-link" href="#">Weather</a>
+          </li>
+
+          <li class="nav-item">
+            <span>Search</span>
+            <i class="fa fa-search"></i>
+          </li>
```
Comment (elmejdki) [inline index.html]: Please change this _li_ tag content into a text field and a button.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1532 @@
...
+            src="https://d.newsweek.com/en/full/1589808/may-22-2020-cover.jpg?w=170&q=95&f=96101d0d430bef83aee5a95169c9b655"
+            alt="last one"
+            class="subimg1"
+          />
+          <img
+            src="https://d.newsweek.com/en/full/1591879/may-29-june-06-2020-cover.jpg?w=170&q=95&f=56aad51c81fc710a32b2778640b7e471"
+            alt="firstone"
+            class="subimg1"
+          />
+          <div class="pre-card">
```
Comment (elmejdki) [inline index.html]: please give some padding to this class so it separate images from text.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1532 @@
...
+          <img
+            src="https://d.newsweek.com/en/full/1589808/may-22-2020-cover.jpg?w=170&q=95&f=96101d0d430bef83aee5a95169c9b655"
+            alt="last one"
+            class="subimg1"
+          />
+          <img
+            src="https://d.newsweek.com/en/full/1591879/may-29-june-06-2020-cover.jpg?w=170&q=95&f=56aad51c81fc710a32b2778640b7e471"
+            alt="firstone"
+            class="subimg1"
+          />
```
Comment (elmejdki) [inline index.html]: Make these two images overlaps over each other as on the original website.
image

File: assets/styles/style.scss [hunk 5]
```diff
@@ -0,0 +1,549 @@
...
+    }
+  }
+
+  .mag-head {
+    text-align: center;
+  }
+
+  .arts {
+    align-items: flex-end;
+    display: flex;
```
Comment (elmejdki) [inline assets/styles/style.scss]: please use Bootstrap flexbox classes instead of custom CSS.

File: assets/styles/style.scss [hunk 6]
```diff
@@ -0,0 +1,549 @@
...
+      color: #333;
+      font-size: 20px;
+      text-align: center;
+    }
+
+    .ft-pad {
+      border: none !important;
+
+      .card-body {
+        padding: 10px 0 !important;
```
Comment (elmejdki) [inline assets/styles/style.scss]: please remove all !important keywords from your CSS, cause it is a really bad practice to override CSS with it.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1532 @@
...
+        </div>
+        <div class="col img-logo">
+          <svg
+            version="1.1"
+            xmlns="http://www.w3.org/2000/svg"
+            xmlns:xlink="http://www.w3.org/1999/xlink"
+            x="0px"
+            y="0px"
+            viewBox="0 0 395.6 60"
+            style="enable-background: new 0 0 395.6 60;"
```
Comment (elmejdki) [inline index.html]: please remove all inline-styling from your HTML use your CSS files instead.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great Work @paulzay :tophat: you need only a few changes to make your project approved :smiley:.
### status: require changes

- give your web page a fixed width so it shows well on big screens like the original one.
- Make sure that you align the search input field with the menu items and also the button.
image

- Don't allow the main navbar links to break on small screens.
image

- Fix the pre-foot div (on the bottom) it gives a horizontal scroll on different breakpoints.
- Fix this overlapping section issues on tablet view:
image
- please add some margin between Images:
image

- Fix the footer to match the original page on mobile view:



### kindly fix all the comments above and below before you submit an other code review.
_If you need anything don't hesitate to contact me on slack, you will find me as **Zakaraie El Mejdki**_

File: index.html (hunk 7 above)
Comment (BrittanyBlake) [reply index.html]: As requested by the previous TSE, please remove in-line styling from your HTML file

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️
****
### Hi @paulzay !!
You have done a great job with this project so far!👍 It looks almost identical to the original and you have done a great job implementing the changes required by the TSE. 🎖️ However, there is one area where you could improve ⬇️
****

- [ ] As requested by the previous TSE, you should give your webpage a fixed width so it shows well on larger screen sizes. I have attached a screenshot of what it currently looks like on my screen 👇
Screen Shot 2020-06-07 at 6 53 53 PM

- [ ] The footer is not the same as the footer on the original page on the small screen size. This was also requested by the previous TSE. I have attached screenshots of your clone as well as the original so that you can compare both.
Your clone 👇 
Screen Shot 2020-06-07 at 8 06 08 PM
The original page 👇 
Screen Shot 2020-06-07 at 8 06 30 PM
****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @paulzay   :handshake: 

### :medal_sports: You are doing a great job :+1:  nice work so far, it looks like the original.

### There some necessary changes before we could approve the project:
- Please implement what @BrittanyBlake asked you to do.
- Your project at very big resolution:
Screenshot from 2020-06-08 11-14-46
- The original project at big resolution:
Screenshot from 2020-06-08 11-20-33
- Also please fix this part and make it as the original:
your project:
Screenshot from 2020-06-08 11-16-27
The original:
Screenshot from 2020-06-08 11-22-05
- Please fix validators errors:
Screenshot from 2020-06-08 11-22-54
### Once you are done, submit for another code review. Good Luck :+1:

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE) 🧐

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🔧 

1. There is a little detail with the JS (that you shouldn't be using)

Screen Shot 2020-06-08 at 7 09 25 AM

2. This section doesn't look similar to the one displayed on the original page (mobile screens)

**Original**

Screen Shot 2020-06-08 at 7 14 43 AM

**Yours**

Screen Shot 2020-06-08 at 7 14 54 AM

3. The footer social network links are too far away from the bottom line (mobile screens)

**Original**

Screen Shot 2020-06-08 at 7 15 21 AM

**Yours** (the line is not right below the links, there is a significant separation)

Screen Shot 2020-06-08 at 7 15 16 AM

4. The **Start your day with our top 5 articles** section has too much width, and the heading's font-weight is too low (tablet screens)

**Original**

Screen Shot 2020-06-08 at 7 19 56 AM

**Yours**

Screen Shot 2020-06-08 at 7 19 49 AM

## What to do now?

Implement the changes, and submit another CRR (Code Review Request)

## How to reach out to me?

I'm available in slack as **srodrig**, happy coding! :wave:

**Review by HADDADSOHAIB — APPROVED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Project Approved :100: 

### Hello @paulzay    :handshake: 

### Good job :clap:  your project satisfy the requirements :medal_sports: and I approve this project

### Congratulations, you can move on and merge this branch.

**Pull request merged**